tests/_test_utils/test_mssautoplot

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+"""
+
+    tests._test_utils.test_mssautoplot
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
the underline is in the reST to indicate a heading, it can be same length or longer. 
If it looks shorter here, it may be because of the font. I don't counted it.